src/App.js

```diff
@@ -1,27 +1,40 @@
-import logo from './logo.svg';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Navigation from './components/Navigation';
+import Login from './components/Login';
+import Signup from './components/Signup';
+import Cars from './components/Car/Cars';
 import './App.css';
+import AddCar from './components/Car/carForm/carForm';
+import CarRemoveList from './components/Car/CarRemoveList';
+import RentsShow from "./components/rents/RentsShow";
+import AddRentForm from "./components/rents/AddRentForm";
+import Detail from "./components/Car/detail/detail";
 
 function App() {
```

Comment:
- [ ] Throughout your app,  the given design has not been followed consistently, which results in misalignments and CSS overflows:

![image](https://user-images.githubusercontent.com/57421823/207289369-c81c2185-cdeb-4083-ade2-b1de70c19dd1.png)

Also on the mobile design:

![image](https://user-images.githubusercontent.com/57421823/207289594-86003fb4-07c4-47b5-9355-ab898a1e6d5f.png)

Kindly try to follow the given design closely, and correct the overflows and misalignments so your app looks more professional. 💪